src/main/java/appeng/tile/misc/CondenserVoidInventory.java

```diff
@@ -19,31 +19,30 @@
 package appeng.tile.misc;
 
 
-import appeng.api.AEApi;
 import appeng.api.config.AccessRestriction;
 import appeng.api.config.Actionable;
 import appeng.api.networking.security.IActionSource;
 import appeng.api.storage.IMEMonitor;
 import appeng.api.storage.IMEMonitorHandlerReceiver;
 import appeng.api.storage.IStorageChannel;
-import appeng.api.storage.channels.IFluidStorageChannel;
-import appeng.api.storage.data.IAEFluidStack;
+import appeng.api.storage.data.IAEStack;
 import appeng.api.storage.data.IItemList;
-import appeng.fluids.util.FluidList;
 
 
-class CondenserFluidInventory implements IMEMonitor<IAEFluidStack>
+class CondenserVoidInventory<T extends IAEStack<T>> implements IMEMonitor<T>
 {
 
 	private final TileCondenser target;
+	private final IStorageChannel<T> channel;
 
-	CondenserFluidInventory( final TileCondenser te )
+	CondenserVoidInventory( final TileCondenser te, final IStorageChannel<T> channel )
 	{
 		this.target = te;
+		this.channel = channel;
 	}
 
 	@Override
-	public IAEFluidStack injectItems( final IAEFluidStack input, final Actionable mode, final IActionSource src )
+	public T injectItems( final T input, final Actionable mode, final IActionSource src )
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Refactor this method to not always return the same value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3516)